<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VSInfoWidget</class>
 <widget class="QWidget" name="VSInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Info Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="settingsTab">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>328</width>
            <height>448</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <property name="spacing">
            <number>4</number>
           </property>
           <item row="1" column="0">
            <widget class="VSVisibilitySettingsWidget" name="visibilityWidget" native="true"/>
           </item>
           <item row="2" column="0">
            <widget class="VSColorMappingWidget" name="colorMappingWidget" native="true"/>
           </item>
           <item row="0" column="0">
            <widget class="VSFilterSettingsWidget" name="filterWidget" native="true"/>
           </item>
           <item row="3" column="0">
            <widget class="VSTransformWidget" name="transformWidget" native="true"/>
           </item>
           <item row="4" column="0">
            <widget class="VSAdvancedVisibilitySettingsWidget" name="advVisibilityWidget" native="true">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="infoTab">
      <attribute name="title">
       <string>Information</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>328</width>
            <height>448</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <property name="spacing">
            <number>4</number>
           </property>
           <item row="0" column="0">
            <widget class="VSDatasetInfoWidget" name="datasetInfoWidget" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VSTransformWidget</class>
   <extends>QWidget</extends>
   <header location="global">VSTransformWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VSFilterSettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">VSFilterSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VSVisibilitySettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">VSVisibilitySettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VSColorMappingWidget</class>
   <extends>QWidget</extends>
   <header location="global">VSColorMappingWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VSAdvancedVisibilitySettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">VSAdvancedVisibilitySettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VSDatasetInfoWidget</class>
   <extends>QWidget</extends>
   <header location="global">VSDatasetInfoWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
